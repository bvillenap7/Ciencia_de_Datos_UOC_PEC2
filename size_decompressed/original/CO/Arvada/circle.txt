10:30  lighted circle over arvada colorado , middle  became   bright and objectfaded away;2011-08-01 22:30:00;30 seconds;10:30  lighted circle over arvada colorado , middle  became   bright and objectfaded away round circle with lights blinking all around , then bottom lights quit and middle lit up.  top half of circle started flashing. object then got bright then faded til it was out of sight in 2 to 3 seconds. quite extrordinary ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.8289982352941;-105.110888235294
5 dancing orbs that disappeared.;2014-07-04 09:20:00;7 minutes;5 dancing orbs that disappeared 5 circle lights that danced changed to 3 in a triangular shape than reapoeared as 5;39.8289982352941;-105.110888235294
Small balls of light changing positions, sharp turns.;2016-02-21 06:50:00;5 minutes;small balls of light changing positions, sharp turns Looking to the west sky.  Many balls of light changing positions, sharp turns, hovering and sort of flying around in circles.  Stationary red lights in area.  Then disappeared.;39.8289982352941;-105.110888235294
5 circular/cylindrical flying objects flying higher than planes;2016-09-04 17:45:00;10 minutes;5 circular/cylindrical flying objects flying higher than planes Witnessed 5 flying objects, circular/ cylindrical in shape. Very high in the sky. Met at one point and then scattered, leaving only one to follow. Light in color.;39.8289982352941;-105.110888235294
