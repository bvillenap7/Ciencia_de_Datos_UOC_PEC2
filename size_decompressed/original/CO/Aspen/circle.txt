Friend and I called the police, something neither of us do unless its important.  ((NUFORC Note:  Sighting of Venus.  PD));2015-10-10 03:00:00;1 hour +;Friend and I called the police, something neither of us do unless its important. Craft flashing green blue red moving in short thrusts.  From Aspen it was seen facing East towards Denver over the Independence Pass.   10x brighter than anything else in the sky. ((NUFORC Note:  Sighting of Venus.  PD));39.1977833333333;-106.825375
