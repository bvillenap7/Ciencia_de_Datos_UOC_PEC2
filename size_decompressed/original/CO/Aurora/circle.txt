i could see it but my friend standing next to me could not it was plain as day right there.;1977-11-20 19:00:00;15 min;i could see it but my friend standing next to me could not it was plain as day right there. the shape of what I saw was circular but like a hockey puck it had round lights around the outside of the round part, the lights were white the light traveled one light at at a time around the round part of the object . the light seemed to skip to the next bulb shutting off the previous bulb as it traveled around and around the outside.this object was not far away at all if it were a plane or hellocopter i would have been able to hear it clearly but there was no sound at all. it moved very slowly and stopped at least once it sharply changed direction at least twice i watched it travel across denver to they edge of the front range then it took off at an angle into the sky like a flash and disappeared. but there is more that happened while it was above. i ran across the street to my friends house. i got him to come and see it i pointed directly at it, it was right there still but he could not see it at all this was very confusing at the time i was ten or eleven years old no one would believe me. also when it stopped for a second i felt like it had seen me watching it then it slowly started moving again. ((NUFORC Note:  Date is approximate.  PD));39.6695444272445;-104.803991486068
4th of July 2013 at 9:40pm Aurora, Co. 3 orange circles  during firework seen by 4 people duration about 5 minutes;2013-07-04 21:40:00;3 minutes;4th of July 2013 at 9:40pm Aurora, Co. 3 orange circles  during firework seen by 4 people duration about 5 minutes 9:40pm 4th of July 2013 in Aurora, Colorado.  3 bright orange circled lights appeared in the denver sky, hovering over the city sky.  They were changing formation often.  First they would be in the formation of a triangle, then they would go into the formation of a straight line.  One time 2 of the objects would rotate around the third one.  Twice it appeared that they would all blend as one object for 2 seconds before breaking into 3 objects again.  There were 4 of us who witness this event and we can all see that there were some rotation within the object itself.  We were all so in awe with what we were seeing.  Anyhow, 2 of the objects disappear so quickly like a flash of light going 500 miles per hour type speed.  The third object hover for a few seconds longer and it just shot up into the air so fast it look like it evaporated.  Amazing, just amazing.  These objects were larger than a helicopter.  Believe me I was in the military for 5 years and I know ! what and how civilian and miliar choppers move and these objects were not either these objects were out of this world;39.6695444272445;-104.803991486068
((HOAX??))  We saw a blue circle go across the sky pretty fast, and was all by itself.;2015-12-21 19:35:00;1 mintute;We saw a blue circle go across the sky pretty fast, and was all by itself. ((NUFORC Note:  Witness provides no contact information.  Hoax??  PD));39.6695444272445;-104.803991486068
((HOAX??))  Large light, changing from green to red, moving erratically with great speed in western sky.;2016-05-28 22:25:00;10 minutes;((NUFORC Note:  When we first posted this report, we assumed that it was sincere, and submitted by a seriou-minded source.  However, that party has failed to respond to all our attempts to communicate with him, so we suspect that it may have been an attempt to hoax our Center.  We are uncertain.  PD)) Large light, changing from green to red, moving erratically with great speed in western sky. Directly due west of Aurora at 22:25, I witnessed what appeared to be a stationary red star, deep red in color with magnitude approximately 3 times that of Jupiter.  Position of red object was nearly 6 o'clock from Jupiter's position.   During observation, after approximately 3 minutes of observing while trying to determine what it was, I was seeing, object shifted to an emerald green color and began drifting - imagine a leaf falling to the ground.  This drift pattern of shifting left to right in gradually increasing arcs continued for approximately one minute before object became stationary again and began shifting back and forth from red to green.  Due to street light, I was unable to focus phone for photograph.  Object remained stationary for several minutes, changing in color with varying speeds from red to green before it moved incredibly fast across the sky due north. Three orb lights moving in different directions converged on the place where object was last stationary forming a very accurate triangle before blinking out.  They just vanished.  Object then raced back across sky for less than 10 seconds, changing colors from red to green while doing so before moving beyond my line of sight due west. As a photographer and amateur astronomer, I've seen meteorites, atmospheric impact on projected magnitude of various stars and planets and I am more than familiar with aircraft and helicopter appearance at night - this was absolutely not any of these things.  I can't tell you what it was that I saw, but I can tell you what it was not and it was not any standard astronomical object or known terrestrial craft. ((NUFORC Note:  We spoke via telephone with this witness, and he sounded to us to be quite serious-minded.  We suspect that he is an excellent witness.  PD));39.6695444272445;-104.803991486068
We saw a bright light traveling E to west, at a moderate speed and elevation that lasted up to 2 min;2016-09-28 20:16:00;2 minutes;While sitting in the back yard.  We saw a bright light traveling east to west, at a moderate speed and elevation, that lasted up to 2 minutes until the light went dim.;39.6695444272445;-104.803991486068
Silver object in the sky moving slowly west to east then fast to north.;2018-05-31 10:00:00;15 minutes;Silver object in the sky moving slowly west to east then fast to north. Silver colored  ball in the sky moving from west to east very slowly.  Once it got near Buckley Air Force Base it turned 90 degrees to the north then sped up dramatically and continued to the north out of sight. ((ADDENDUM FROM WITNESS)) The object was star like in shape. Also, I work for ((deleted)) and I don’t want to expose myself to too much media attention.  But I did want to report the sighting to you.  If you do happen to talk about the report, just don’t use my name.  Thanks. ((END ADDENDUM));39.6695444272445;-104.803991486068
A large bright circle in the sky over Aurora;2020-03-23 21:15:00;1.5 hours;A large bright circle in the sky over Aurora A large bright circle in the sky hung motionless for a very long time, without losing brightness, for an hour and a half, in the north-west direction.  Photos and videos were taken.;39.6695444272445;-104.803991486068
