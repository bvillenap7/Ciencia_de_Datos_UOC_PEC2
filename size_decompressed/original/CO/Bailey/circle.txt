Myself and a friend witnessed a bright, white cirle of light about 20 yds across.It was about 500 feet above us.Everything went silent.;1998-06-06 00:30:00;5 minutes;Myself and a friend witnessed a bright, white cirle of light about 20 yds across.It was about 500 feet above us.Everything went silent. We were sitting on the hod of a car talking when my friend yelled Look at that. I turned and directly overhead was a bright white circle of light about 20 yards across. It had a faint red halo around it, outside of which the white light was dimmer. The object had a tail. The tail was a deep blue. Within it were diamond aquamarine shapes which fluctuated in color and position. There had been clouds in the sky prior to the event. The sky was completely clear. There was a noisy fountain behind us. Throughout the entire event we heard nothing. All sound stopped. The obeject slowly flew over us about 500 feet up in relation to tree tops. Both myself and the witness felt completely at peace. We both had an out of body experience. As it flew over the object stayed a perfect circle. It did not elipse. In total the event seemed to take about five minutes. Later we discovered it had been about 20 minutes. The object came to the tree line and disapeared. It never sped up. We rushed to look around the trees and saw nothing. It was gone instantaneously. It wasn't as if the light had blinked out. It was simply no longer there.We rushed inside to tell two other friends. They did not witness the light. They came outside with us and we all looked around. The four of us saw two planes flying by. There are no commercial flight paths over this area. We had all worked and lived in the area for at least two years and know the skies pretty well from being outdoors so much. The whole experience felt very peaceful during but afterward both my friend and I were terrified of seeing it alone. Myself and my friend are both 22 years old and college students.;39.4497;-105.4692
Red and blue flashing white moving round objects.;2015-09-27 22:30:00;20 minutes;red and blue flashing white moving round objects. While watching the lunar eclipse, my brother and saw orange/blue  blinking round object cutting across the west part of the sky going faster than plane,  from us.satellite .shooting star etc....., it then slowed up and when straight away f trom us.....then text time we saw it , it was way 1 ... 0'x farther away, a very small dot of light but was moving from right to left......it was very fast because it was so far away but it hooked up with another white light changing colors.....then both of them started to zig zag as if they were having fun.....after the moon eclipse was over they both flew off.......;39.4497;-105.4692
5 lights seen travelling in a straight line followed shortly later by 2 similar lights.  ((Starlink satellites));2020-03-26 08:53:00;5 minutes;5 lights seen travelling in a straight line followed shortly later by 2 similar lights. At approximately 8:53PM my neighbor, a police officer, and I witnessed UFO phenomena in the mountains 45 minutes Southwest of Denver.  If you are familiar with what a satellite looks like when going across a sky it will be easier to understand what we saw.  Essentially it looks like a star that slowly creeps it's way across the sky.  I was outside, bored from quarantine and playing with my badass blue light laser from China. It is like holding a 25 mile long lightsaber.  I was pointing out constellations for my neighbor when all of a sudden I saw what I assumed to be a satellite.  I circled it with the beam and showed to identify it.  A few seconds later we saw another, directly in line. Strange but not crazy...  Except they kept coming.  Five in a row forming a perfect straight line streaking the sky. They were evenly spaced, but strangely, had differnt magnitudes of brightness.  I freaked the fuck out and tried to take a video, but as you know you can't tape things with the brightness of stars using a cell phone camera.  After the objects made their way across the horizon, two more seemed to follow quickly in pursuit, not in a line but side by side. These however were the same brightness as each other and bluish in color. Also unlike the previous five, these two disappeared suddenly after flying half way across the sky.  From our position we witnessed all objects follow a Northeastern trajectory.  Following the incident I called a Denver new station and asked if they had other reports. They did and dismissed them as satellites. I am an amateur astronomer and know what I saw. Satellites do not travel in formation.  I believe the objects did not have physical lights on them but were rather travelling in space. The light from the sun made their reflections visible. This rules out military aircraft which as a dime a dozen in CO. And not to mention they were completely silent.;39.4497;-105.4692
