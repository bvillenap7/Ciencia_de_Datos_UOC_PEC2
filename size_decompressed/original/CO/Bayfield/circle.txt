Saw same object as people in Washington on 22nd.  Western Sky.;2002-03-29 19:00:00;30 minutes;Saw same object as people in Washington on 22nd.  Western Sky. Saw the same thing as reported by the Washington people.  6 spheres on outside of 1 middle sphere.  Had blue green color.  Looked at through spotting scope in western sky.;37.261;-107.6068
