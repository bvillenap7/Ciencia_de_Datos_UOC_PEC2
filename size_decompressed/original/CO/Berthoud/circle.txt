Circular shapes crafts with for Bright Lights;2019-05-14 05:30:00;10 minutes;circular shapes crafts with for Bright Lights Reported circular shaped craft with for Big Orange lights seeming to carry the craft along in a lightning storm;40.2993;-105.1055
