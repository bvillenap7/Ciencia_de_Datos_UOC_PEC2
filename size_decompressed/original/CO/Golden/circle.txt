We saw a bright large white light in the sky that was moving and then disappeared.;2006-03-14 20:00:00;40 seconds;We saw a bright large white light in the sky that was moving and then disappeared. We were walking in the neighborhood with our dogs and one of the dogs stopped. We noticed a huge white yellowish light in the sky. It was way bigger than any planet or star, but too far away to tell detail. It looked like a street lamp was suspended in the air. It was moving around, in a straight direction. It went behind a cloud but we could still see some of it and then it disappeared.;39.7598265306123;-105.220002040816
A bright light dropped out of the bucket of the Big Dipper. It went from the South to the North then disappeared.;2013-05-26 21:44:00;5 minutes;A bright light dropped out of the bucket of the Big Dipper. It went from the South to the North then disappeared. My husband and myself were sitting outside on our deck.  We looked up at the Big Dipper and out of the bucket dropped a bright light that went from South to North across the bucket.  Then it disappeared.;39.7598265306123;-105.220002040816
Blue/purple orb seen from North Table in Golden, CO.  ((anonymous report));2017-03-27 01:30:00;30 minutes;Blue/purple orb seen from North Table in Golden, CO. I stepped out onto my front porch and looked into the northeast sky.  I noticed what looked like a blue/purple orb flying in a T shaped pattern.  It flew very slowly, and very directly.  When it changed direction, it did so immediately.  When it first caught my eye around 1am, it was moving to the left, then headed down.  Headed back up, then moved left again.  Stopped for a brief minute, and when it did, it became a semi-solid/pulsating randomly white light.   When it began moving again to the right, it became the blue/purple orb it had been when it first caught my eye.  After moving for a brief moment it stopped, changed to a white light again, and this is where it sat for a solid 20mins.  Then, out of nowhere, it became the blue/purple orb once again, this time moving straight up for a split second, moving significantly faster, and then it was gone. ((NUFORC Note:  Source of the report elects to remain totally anonymous provides no contact information.  PD));39.7598265306123;-105.220002040816
